src/backlight/portfolio/portfolio.py

```diff
@@ -90,14 +90,98 @@ def construct_portfolio(
     return Portfolio(positions)
 
 
-def calculate_pl(pt: Portfolio, mkt: List[MarketData]) -> pd.DataFrame:
+# def calculate_pl(pt: Portfolio, mkt: List[MarketData]) -> pd.DataFrame:
+#     """
+#     Apply the sum on the homogenized portfolio
+#     args:
+#         - portfolio : a defined portfolio
+#         - mkt : list of marketdata for each asset
+#         - base_ccy : asset of reference
+
+#     """
+#     pl = pt.value()
+#     return pl.sum(axis=1)
+
+
+def homogenize_pl(
```

Comment:
I think we do not need to have the `homogenize_pl`.

Please rename function as `calculate_pl`.

```
    pl = portforlio.value
    symbols = portforlio.symbols
    for symbol in symbols:
        if xxxx:
            pls.loc[:, symbol] = convert_positions(pls.loc[:, symbol])

    return pl
```

Reply:
portfolio.value() function return the pl of each asset, while the convert_positions function takes a whole position. Do we need to apply the conversion+sum only to the pl of each asset? Thanks!